test/library/packages/Sort/RadixSort/radixMSBModule.chpl

```diff
@@ -0,0 +1,83 @@
+ use BitOps;
+ use Sort;
+ use Random;
+ config param bucketBits=8; 
+
+ module MSBmodule {
```

Comment:
Not sure how much it matters, since we intend for this to end up in the `Sort` module eventually, but we generally don't include the name `module` in a module name. 

`Radix`, `MSB` or `RadixMSB` might be a better fit.